### PR TITLE
AST-289 - refactor: Stick footer at the bottom as we have removed default sidebar

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Stick footer at the bottom as we have removed default sidebar using CSS.

### Types of changes
- Improvement

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by premanshup — APPROVED** [review]

**Comment by imnavanath** [comment]
@sagarb90 do we need to add a changelog here? I think it will be easy to track while merging it into next-release and update release.

**Comment by sagarb90** [comment]
> @sagarb90 do we need to add a changelog here? I think it will be easy to track while merging it into next-release and update release.

Yes we need for tracking purpose so I have added changelog for it.

File: assets/css/unminified/frontend-rtl.css [hunk 1]
```diff
@@ -3227,6 +3227,29 @@ input[type="color"]::-webkit-color-swatch {
   }
 }
 
+@media (min-width: 768px) {
```
Comment (imnavanath) [inline assets/css/unminified/frontend-rtl.css:3230]: @sagarb90 admin-bar has 782px breakpoint, I think should be update likewise, please verify once - https://share.bsf.io/Wnuxpd59

Comment (imnavanath) [reply assets/css/unminified/frontend-rtl.css]: Removed CSS code entirely

**Comment by sagarb90** [comment]
This PR will be required in the future as this is related to the removed default sidebar and refactor default blog page design

**Pull request closed without merging**